.travis.yml

```diff
@@ -47,5 +52,6 @@ deploy:
     file: "netdata*.tar.*"
     on:
       condition: $CC = gcc
+      condition: $TRAVIS_OS_NAME = linux
```

Comment:
I guess it wants merging with the line above e.g. with &&, otherwise multiple builds will try to deploy their source files.